Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -986,6 +986,7 @@ theorem eLpNorm'_eq_zero_iff (hq0_lt : 0 < q) {f : α → ε} (hf : AEStronglyMe
     eLpNorm' f q μ = 0 ↔ f =ᵐ[μ] 0 :=
   ⟨ae_eq_zero_of_eLpNorm'_eq_zero (le_of_lt hq0_lt) hf, eLpNorm'_eq_zero_of_ae_zero hq0_lt⟩
 
+variable {ε: Type*} [ENorm ε] in
```

Comment:
A future PR will audit the enorm lemmas in this file for unnecessarily general hypotheses. (That work has been performed in #24352 already, I just need to PR it.)